spec/spec_helper.rb

```diff
@@ -6,14 +6,27 @@
 
 require "factory_bot"
 require "simplecov"
+require "simplecov-lcov"
 require "webmock/rspec"
 require "fixtures/organizations"
 require "fixtures/collections"
 require "fixtures/large_clusters"
 require "pry"
 require "settings"
 require "services"
+require "sidekiq/testing"
+
+SimpleCov::Formatter::LcovFormatter.config do |c|
```

Comment:
Is this related to a particular issue or did you just want to have better coverage analysis?

Reply:
Not related to any particular issue -- mainly just wanted to be able to see coverage reports in pull requests and to be able to see coverage over time.